common/src/main/res/values/strings.xml

```diff
@@ -1008,6 +1008,20 @@
     <string name="widget_config_action_error">A custom component is preventing action data from loading.</string>
     <string name="widget_creation_error">Unable to create widget.</string>
     <string name="widget_entity_fetch_error">Unable to fetch data for configured entity.</string>
+    <string name="grid_widget_actions">Actions</string>
+    <string name="widget_grid_add_action">Add action</string>
+    <string name="widget_grid_desc">A configurable grid of actions</string>
+    <string name="widget_grid_domain">Domain</string>
+    <string name="widget_grid_edit_action">Edit action</string>
+    <string name="widget_grid_entity">Entity</string>
+    <string name="widget_grid_entity_state_unknown">Unknown</string>
+    <string name="widget_grid_icon">Icon</string>
```

Comment:
Some of these strings appear to be duplicated, we should re-use the same string if we have them to prevent duplicate translations.